metpy/calc/indices.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2008-2015 MetPy Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+"""Contains calculation of various derived indicies."""
+import numpy as np
+
+from . import mixing_ratio, saturation_vapor_pressure
+from .tools import get_layer
+from ..constants import g, rho_l
+from ..package_tools import Exporter
+from ..units import units
+
+exporter = Exporter(globals())
+
+
+def precipitable_water(dewpt, p, top=400 * units('hPa')):
```

Comment:
It would be rather uncommon, but someone could want to see how much precipitable water some storms elevated over a stable layer had to work with or something like that. I don't think I've ever actually seen precipitable water plotted starting above the surface though.